src/sagas/deposit.js

```diff
@@ -301,6 +304,50 @@ export function* getBitcoinAddress() {
   yield* autoSubmitDepositProof()
 }
 
+function createConfirmationChannel(deposit) {
+  return eventChannel((emit) => {
+    const listener = ({
+      transactionID,
+      confirmations,
+      requiredConfirmations,
+    }) => {
+      emit({ transactionID, confirmations, requiredConfirmations })
+
+      // Close channel once we have all the required confirmations
+      if (confirmations === requiredConfirmations) {
+        emit(END)
+      }
+    }
+
+    deposit.onReceivedFundingConfirmation(listener)
+
+    // no-op
+    // eventChannel subscriber must return an unsubscribe, but tbtc.js does not
+    // currently provide one
+    return () => {}
```

Comment:
What structure is this unsubscribe in? We can probably add this in future work.

Reply:
The `eventChannel` would want something like:
```
    const unsubscribe = () => {
      deposit.removeReceivedConfirmationHandler( listener)
    }

    return unsubscribe
```